collectors/o365/o365_collector.js

```diff
@@ -174,6 +174,17 @@ class O365Collector extends PawsCollector {
                 try {
                     let error = JSON.parse(message.slice(message.indexOf('{'), message.lastIndexOf('}') + 1));
                     err.errorCode = error.error;
+                    if (error.error_codes) {
```

Comment:
Do we want to return the error code in the message (if customer/support want to search on the error code)?

Reply:
okay will do that.

Reply:
Done